OracleDatabase/RAC/OracleConnectionManager/README.md

```diff
@@ -1,30 +1,31 @@
-# Oracle Connection Manager on Docker
-Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle Database please see the [Oracle Connection Manager online Documentation](http://docs.oracle.com/en/database/).
+# Oracle Connection Manager on Container
+Sample container build files to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle Database please see the [Oracle Connection Manager online Documentation](http://docs.oracle.com/en/database/).
 
 ## How to build and run
-This project offers sample Dockerfiles for:
+This project offers sample container files for:
+  * Oracle Database 21c Client (21.3) for Linux x86-64
   * Oracle Database 19c Client (19.3) for Linux x86-64
   * Oracle Database 18c Client (18.3) for Linux x86-64
   * Oracle Database 12c Release 2 Client (12.2.0.1.0) for Linux x86-64
 
-To assist in building the images, you can use the [buildDockerImage.sh](dockerfiles/buildDockerImage.sh) script. See section **Create Oracle Connection Manager Image** for instructions and usage.
+To assist in building the container images, you can use the [buildContainerImage.sh](dockerfiles/buildContainerImage.sh) script. See section **Create Oracle Connection Manager Image** for instructions and usage.
 
-**IMPORTANT:** Oracle Connection Manager container is useful when you want to bind single port to host and serve many container on different ports. Oracle Connection manager provide proxy conections. If you are running Oracle RAC Database on docker/container and network is not available for users, you can use Oracle Connection Manager image to proxy the connections.
+**IMPORTANT:** Oracle Connection Manager container is useful when you want to bind single port to host and serve many container on different ports. Oracle Connection manager provide proxy connections. If you are running Oracle RAC Database on docker/container and network is not available for users, you can use Oracle Connection Manager image to proxy the connections.
```

Comment:
```suggestion
Oracle Connection Manager binds to a single port on your host and proxies incoming connections to multiple running containers. It can also proxy connections for users to Oracle RAC Databases running on internal container networks.
```

Reply:
Issue is fixed, please review.

Reply:
It doesn't appear to have changed at all?

Reply:
Added a fix in a new commit "Added fix suggested for the pull request #2336"